floyd/model/access_token.py

```diff
@@ -9,7 +9,7 @@
 
 def assert_token_not_expired(token):
     payload_base64 = token.split('.')[1] + '==='
-    payload = json.loads(base64.decodestring(payload_base64))
+    payload = json.loads(base64.decodestring(payload_base64.encode('ascii')).decode('ascii'))
```

Comment:
Is this the change for py3?

Reply:
yes, this api in py3 expects bytes instead of string